packages/affiliates/libs/processors.js

```diff
@@ -150,19 +168,15 @@ function EmpBalances(handlers = {}, { collateral, tokens } = {}) {
     // looking at the emp code, i think anyone can call this even if they never had a position
     // this means balances may not exist or may go below 0. we should just catch those errors and ignore
     SettleExpiredPosition(caller, collateralReturned, tokensBurned) {
-      if (collateral.has(caller)) {
-        try {
-          collateral.sub(caller, collateralReturned.toString());
-        } catch (err) {
-          // ignore balance below 0 error
-        }
+      try {
+        collateral.sub(caller, collateralReturned.toString());
+      } catch (err) {
+        // ignore balance below 0 error
```

Comment:
Is this error thrown by the `collateral.sub()` method? Isn't this a BN class? I'm wondering if it'd be better to change that method to just not throw on negative numbers

Reply:
when instantiating the balances class, i have an option to not throw on negative balances, which is set to true. I can probably get rid of this try catch completely now.

Reply:
oh btw its not a BN class, its a custom model "Balances" defined in models.js.